tools/api.py

```diff
@@ -1057,9 +1062,11 @@ def add_cpp_class(self) -> None:
     ${output_decls}
     if (${fname} == nullptr)
         throw std::runtime_error("${name} function is missing.");
+    const char* exception_msg = " ";
```

Comment:
This should be an array: `std::array<char, 256> exception_msg` so it can copy the message.